sdk/communication/communication-sms/samples/v1/typescript/src/usingAadAuth.ts

```diff
@@ -19,8 +19,8 @@ export async function main() {
 
   // You will need to set this environment variable or edit the following values
   const endpoint =
-    process.env.COMMUNICATION_ENDPOINT ||
     parseConnectionString(process.env.COMMUNICATION_SAMPLES_CONNECTION_STRING!).endpoint ||
+    process.env.COMMUNICATION_ENDPOINT ||
```

Comment:
do we still need to keep this line? does it make sense to use the dynamic endpoint?